src/libs/vehicle/ardupilot/ardupilot.ts

```diff
@@ -82,6 +82,8 @@ export abstract class ArduPilotVehicle<Modes> extends Vehicle.AbstractVehicle<Mo
   onMAVLinkMessage = new SignalTyped()
   _flying = false
 
+  protected currentSystemId: number
```

Comment:
You can make it default to 1 like this:

```typescript
protected currentSystemId = 1
```

Typescript will set the value to 1 and infer automatically that the type is `number`.